ios/FluentUI.Demo/FluentUI.Demo/ColoredBackgroundView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+public enum ColoredBackgroundStyle: Int {
```

Comment:
feels like the naming is too generic when only specific case would use this class and style.

Reply:
That's fair. It probably also doesn't need to be public. I'll see if I can come up with a better name.